client/signup/jetpack-connect/index.jsx

```diff
@@ -201,12 +206,24 @@ const JetpackConnectMain = React.createClass( {
 							text={ this.translate( 'Once the plugin is activated you\'ll click this green \'Connect\' button to complete the connection.' ) }
 							example={ <JetpackExampleConnect url={ this.state.currentUrl } /> } />
 					</div>
+					{ this.renderBackButton() }
 					<Button onClick={ this.installJetpack } primary>{ this.translate( 'Install Jetpack' ) }</Button>
 				</div>
 			</Main>
 		);
 	},
 
+	renderBackButton() {
+		return (
+			<a className="navigation-link jetpack-connect__back-button" onClick={ this.clearUrl }>
```

Comment:
sooner or later (well, this week supposedly) we are going to add tracking, and clearUrl would both call dismissUrl & the tracking methods